src/lizaalert/courses/admin.py

```diff
@@ -59,8 +59,12 @@ class ChapterInline(admin.TabularInline):
 
     # Метод для отображения ссылки на главу курса
     def get_chapter_link(self, obj):
-        url = reverse("admin:courses_chapter_change", args=(obj.id,))
-        return format_html('<a href="{}">{}</a>', url, obj.title)
+        id = obj.id
+        if not id:
+            return "Нет названия"
+        title = obj.title or "Нет названия"
+        url = reverse("admin:courses_chapter_change", args=(id,))
+        return format_html('<a href="{}">{}</a>', url, title)
```

Comment:
ок, давай title определим в одном месте

```suggestion
        title = obj.title or "Нет названия"
        if not (id := obj.id):
            return format_html('<span>{}</span>', title)
        return format_html('<a href="{}">{}</a>', reverse("admin:courses_chapter_change", args=(id,)), title)
```